sample-app/src/components/VisualAutocomplete/VisualSearchBar.tsx

```diff
@@ -12,9 +12,12 @@ import useSearchWithNearMeHandling from '../../hooks/useSearchWithNearMeHandling
 import { builtInCssClasses, SearchBarCssClasses } from '../SearchBar';
 import { CompositionMethod, useComposedCssClasses } from '../../hooks/useComposedCssClasses';
 import { ReactComponent as YextLogoIcon } from '../../icons/yext_logo.svg';
+import { ReactComponent as RecentSearchIcon } from '../../icons/history.svg';
 import renderAutocompleteResult from '../utils/renderAutocompleteResult';
+import RecentSearches from "recent-searches";
 
 const SCREENREADER_INSTRUCTIONS = 'When autocomplete results are available, use up and down arrows to review and enter to select.'
+const DEFAULT_RECENT_SEARCH_KEY = '__yxt_recent_searches__'
```

Comment:
default implies this can be overridden but since it can't, maybe just RECENT_SEARCHES_KEY?